core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/LogicalOp.scala

```diff
@@ -219,6 +220,7 @@ trait StateTransferFunc
     new Type(value = classOf[JavaUDFOpDesc], name = "JavaUDF"),
     new Type(value = classOf[RUDFOpDesc], name = "RUDF"),
     new Type(value = classOf[RUDFSourceOpDesc], name = "RUDFSource"),
+    new Type(value = classOf[ScorerOpDesc], name = "Scorer"),
```

Comment:
`scorer` itself is not meaningful enough. call it `MachineLearningScorer`?

Reply:
fix